systemtest/src/test/resources/oauth2/create_realm_authorization.sh

```diff
@@ -0,0 +1,676 @@
+#!/usr/bin/env bash
+
```

Comment:
A comment to say that this script is used to setup the Oauth tests would be a good idea.